content/docs/install/plugins.md

```diff
@@ -46,3 +46,18 @@ autocmd! BufNewFile,BufRead Dvcfile,*.dvc,dvc.lock setfiletype yaml
 ```
 
 to your `~/.vimrc`(to be created if it doesn't exist).
+
+## Doom Emacs
+
+In order to recognize `dvc.lock` and `.dvc` files as YAML in Doom Emacs, you
+should add:
+
+```emacs-lisp
```

Comment:
This isn't a supported highlighter language though. Maybe just remove it unless there's something else that seems appropriate from https://prismjs.com/#supported-languages @roryk ?

Reply:
emacs-lisp is supported, it's on there under lisp.

Reply:
Ah yes, I didn't see it for some reason. OK all good then